config/application.rb

```diff
@@ -28,12 +28,18 @@
 
 module FindDataBeta
   class Application < Rails::Application
-    # Settings in config/environments/* take precedence over those specified here.
-    # Application configuration should go into files in config/initializers
-    # -- all .rb files in that directory are automatically loaded.
-
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 6.0
+
+    # Make `form_with` generate non-remote forms.
+    config.action_view.form_with_generates_remote_forms = false
```

Comment:
Does the app use `form_with`? If not doesn't seem much point in having this config.

Reply:
this was added by someone else on a previous Rails update (3d421d6). I just moved it here to keep stuff centralized. The app broke when I tried to remove it.